docs/reports/2026-05-19-issue-737-historyspice-vocabulary-coverage.md

```
@@ -0,0 +1,907 @@
+# Issue 737 HistorySpice Vocabulary Coverage
+
+Date: 2026-05-19
+
+## Scope
+
+This report tracks the direct dictionary-key coverage dimension added to Issue
+#737. It is independent from route ownership: a route may call the intended
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)